modules/global-classes/global-classes-rest-api.php

```diff
@@ -72,9 +72,9 @@ private function register_routes() {
 				'methods' => 'PUT',
 				'callback' => fn( $request ) => $this->route_wrapper( fn() => $this->put( $request ) ),
 				'validate_callback' => function( \WP_REST_Request $request ) {
-					[ $is_valid ] = Style_Validator::make( Style_Schema::get() )
+					[ $is_valid ] = Style_Parser::make( Style_Schema::get() )
 						->without_id()
-						->validate( $request->get_body_params() );
+						->parse( $request->get_body_params() );
```

Comment:
Feels like we should have `validate` and `sanitize` methods on the `Style_Parser` and also have `parse`.

+ we should add `sanitize_callback` to the routes params.